index.html

```diff
@@ -1024,16 +1024,16 @@ <h2>Extract response headers</h2>
       Given a <a>network request</a> (<var>request</var>) and its corresponding
       <a>response</a> (<var>response</var>), this algorithm generates a report
       about <var>request</var> if instructed to by any matching <a>NEL
-        policy</a>, and queues it for delivery.

```

Comment:
Are there any current callers who need to change to explicitly deliver the network error log?

Reply:
I think there is no spec which explicitly calls this algorithm.

@dcreager 
Is my understanding correct?

Reply:
That’s right, yours is the first.  We should add some language to the NEL spec monkey-patching Fetch for normal requests, but we haven’t done that yet, so there aren’t any other callers to worry about. (That I’m aware of!)